config/systems.yaml

```diff
@@ -1,41 +1,50 @@
-
 systems:
-- name: kind
-  aliases: [kind-kind]
-  overlays: [kind, overlays/kind]
-  master: kind-control-plane
-  workers: [kind-worker]
-  rabbits:
-    kind-worker2: {0: compute-01, 1: compute-02, 6: compute-03}
-    kind-worker3: {4: compute-04}
-- name: craystack-default
-  aliases: [craystack]
-  overlays: [craystack]
-  master: k3s-master
-  workers: [k3s-master]
-  rabbits:
-    rabbit-01: {0: compute-01}
-    rabbit-02:
-    rabbit-03:
-- name: rabbit-dev
-  aliases: [dp0, rabbit-dev-01, rabbit-lab]
-  overlays: [dp0,overlays/rabbit]
-  master: rabbit-dev-vm-k8s-master
-  workers: [rabbit-dev-cn-02]
-  rabbits:
-    rabbit-dev-01: {0: rabbit-dev-cn-01, 1: rabbit-dev-cn-02, 2: rabbit-dev-cn-03, 3: rabbit-dev-cn-04}
-- name: rabbit-htx
-  aliases: [dp1, dp1a]
-  overlays: [dp0,overlays/rabbit]
-  master: rabbit-k8s-master
-  workers: [rabbit-k8s-worker]
-  rabbits:
-    rabbit-node-0: {0: compute-node-0, 1: compute-node-1}
-    rabbit-node-1: {0: compute-node-3, 1: compute-node-2}   # These computes are physically swapped
-- name: rabbit-htx-2
-  aliases: [htx-2]
-  overlays: [dp0,overlays/rabbit]
-  master: compute-node-4
-  workers: [compute-node-5]
-  rabbits:
-    rabbit-node-2: {0: compute-node-4, 1: compute-node-5}
+  - name: kind
+    aliases: [kind-kind]
+    overlays: [kind, overlays/kind]
+    master: kind-control-plane
+    workers: [kind-worker]
+    rabbits:
+      kind-worker2: { 0: compute-01, 1: compute-02, 6: compute-03 }
+      kind-worker3: { 4: compute-04 }
+    overrides:
+      managers: [kind-control-plane]
```

Comment:
What does "manager" mean to you?

Reply:
cray.(nnf|wlm).manager=true

Reply:
The nodes in the 'rabbits' list should be labeled with cray.nnf.node=true and tainted.  The nodes in the 'worker's list get the other label.

That should work for everything.  You shouldn't need the "overrides" section.

Reply:
Agreed, and that is the default behavior, but current init.sh has some exceptions to that rule
dp0: worker nodes are compute nodes
craystack: worker nodes are master (control-plane) nodes

Reply:
The current systems.yaml for craystack shows this:
```
- name: craystack-default
  aliases: [craystack]
  overlays: [craystack]
  master: k3s-master
  workers: [k3s-master]
```
listing the k3s-master as both master and worker.  So there shouldn't be a need for an "override" section.

Reply:
Now I'm catching up with you. Agreed - no need for an override section.

dp0 is _fine_ with just cn-02 as a worker, we could add cn-01 to systems.yaml. 
cn-03 is down and cn-04 is the master so we can ignore those two

Reply:
So for kind, if we get rid of overrides and then set `workers: [kind-control-plane]`, then `kind-worker` is unaccounted for. Do we make that a rabbit?

Reply:
My experience with Kubernetes v1.25.2 was cert-manager needs an untainted pod as the worker; so we should leave the workers value, `workers: [kind-worker]`, alone.